umh.docs.umh.app/content/en/docs/features/postgresql_and_grafana_alerts.md

```diff
@@ -0,0 +1,245 @@
+   and therefore not part of this tutorial.
+
+4. **Configure Notification Policies**
+
+     A notification policy page allows you to link alerts with a contact 
+     point. Notification policies can be added in the **Notification policies**
+     section in the Grafana alerting page, next to **Contact points**.
+
+     - Select **+ New specific policy** to create a new policy, followed by 
+       **+ Add matcher** to choose the label and value from the alert
+       (for example `team = operator`). In this example, both alert1 and 
+       alert3 will be forwarded to the associated contact point. You can 
+       include multiple labels in a single notification policy.
+     - Choose the contact point designated to receive the alert notifications.
+     - Press **Save policy** to finalize your settings. Your new policy will
+       now be displayed in the list.
+
+5. **Configure Mute Timing and Silences**
+
+   To avoid receiving alerts during specific, recurring time periods, create a
+   mute timing that allows you to designate the exact time span when alerts 
+   will be muted. To set up a mute timing, remain in the 
+   **Notification policies** section.
+   - Select **+ Add mute timing** below the notification policies.
+   - Choose a name for the mute timing.
+   - Specify the time during which notifications should not be forwarded.
+     - Time has to be given in UTC time and formatted as HH:MM. Use **06:00** 
+       instead of **6:00** to avoid an error in Grafana.
+   - You can combine several time intervals into one mute timing by clicking 
+     on the **+ Add another time interval** button at the end of the page. 
+   - Click **Submit** to save your settings.
+   - To apply the mute timing to a notification policy, click **Edit** on the
+     right side of the notification policy, and then select the desired mute 
+     timing from the drop-down menu at the bottom of the policy. Click on 
+     **Save Policy** to apply the change.
+   
+   You can also add **Silences** for a specific time frame and labels, in case 
+   you only want to mute alerts once. To add a silence, switch to the 
+   **Silences** section, next to **Notification policies**.
+   - Click on **+ Add Silence**.
+   - Specify the beginning for the silence and its duration.
+   - Select the labels and their values you want silenced.
+   - If needed, you can add a comment to the silence.
+   - Click the **Submit** button at the bottom of the page.
+
+
+The alert is now completely set up and running. In case you followed the 
+tutorial with the simulated data, you should be receiving an alert in Discord
+or the service you chose to receive the alerts. An alert is only sent out once 
+after beeing triggert. For the next alert, it has to return to the normal state,
+so the data no longer violates the rule.
+
+## What are the limitations?
+
+It can be complicated to select and manipulate the desired values to create
```

Comment:
Sounds like ChatGPT again. Please use actual limitations such as:
- not able to alert on results of the API feature, can only alert on whats in the database

think of how you would handle data arriving late. will the alert still work?

Reply:
the main problem i actually the funktionality of Section2 where you formulate the rule and the SQL command. The main problem is described in the article, that grafana can not differentiate between alerts from the same "sensor" with just a different timestamp. Therefore you have to be carefull, when creating the rule. When the data is late, or not there, Grafana will jsut do nothing, as far as I understood it. 

I will change the text a bit to be more specific and add the limitation with the database/ API